lesson1/test/Token.ts

```diff
@@ -0,0 +1,96 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { Contract } from "ethers";
+
+describe("Token", function () {
+  let owner: SignerWithAddress;
+  let otherUser: SignerWithAddress;
+  let token: Contract;
+  const totalSupplyAlsoInitial = "100000000000000000000000";
```

Comment:
Takie rzeczy strasznie się czyta. Dlatego warto jednak ubierać to ładnie np w `BigNumber("100000").mul(BigNumber.from("10").pow(decimals))`